src/pr-info.ts

```diff
@@ -582,70 +603,21 @@ function getCIResult(headCommit: PR_repository_pullRequest_commits_nodes_commit)
-            popularityLevel = "Popular";
-        }
-    }
-    return popularityLevel;
+function downloadsToPopularityLevel(monthlyDownloads: number): PopularityLevel {
+    return monthlyDownloads > CriticalPopularityThreshold ? "Critical"
+        : monthlyDownloads > NormalPopularityThreshold ? "Popular"
+        : "Well-liked by everyone";
 }
 
-function getPopularityLevelFromDownloads(downloadsPerPackage: Record<string, number>) {
-    let popularityLevel: PopularityLevel = "Well-liked by everyone";
-    for (const packageName in downloadsPerPackage) {
-        const downloads = downloadsPerPackage[packageName];
-        if (downloads > CriticalPopularityThreshold) {
-            // Can short-circuit
-            return "Critical";
-        } else if (downloads > NormalPopularityThreshold) {
-            popularityLevel = "Popular";
-        }
-    }
-    return popularityLevel;
-}
-
-interface OwnerInfo {
-    newPackages: string[];
-    allOwners: string[];
-    error: string | undefined;
-}
-
-async function getOwnersOfPackages(packages: readonly string[], fetchFile: typeof defaultFetchFile): Promise<OwnerInfo> {
-    const allOwners: Set<string> = new Set();
-    let newPackages = [], error = undefined;
-    for (const p of packages) {
-        let owners;
-        try {
-            owners = await getOwnersForPackage(p, fetchFile);
-        } catch (e) {
-            error = `error parsing owners: ${e.message}`;
-            break;
-        }
-        if (owners === undefined) {
-            newPackages.push(p);
-        } else {
-            owners.forEach(o => allOwners.add(o));
-        }
-    }
-    return { error, allOwners: [...allOwners], newPackages };
-}
-
-async function getOwnersForPackage(packageName: string, fetchFile: typeof defaultFetchFile): Promise<string[] | undefined> {
-    const indexDts = `master:types/${packageName}/index.d.ts`;
+async function getOwnersOfPackage(packageName: string, version: string, fetchFile: typeof defaultFetchFile): Promise<string[] | null | Error> {
+    const indexDts = `${version}:types/${packageName}/index.d.ts`;
     const indexDtsContent = await fetchFile(indexDts, 10240); // grab at most 10k
-    if (indexDtsContent === undefined) return undefined;
-    const parsed = HeaderParser.parseHeaderOrFail(indexDtsContent);
-    return parsed.contributors.map(c => c.githubUsername).filter(notUndefined);
-}
-
-function noNulls<T>(arr: ReadonlyArray<T | null | undefined> | null | undefined): T[] {
-    if (arr == null) return [];
-    return arr.filter(arr => arr != null) as T[];
+    if (indexDtsContent === undefined) return null;
+    let parsed: HeaderParser.Header;
+    try {
+        parsed = HeaderParser.parseHeaderOrFail(indexDtsContent);
+    } catch (e) {
+        if (e instanceof Error) return new Error(`error parsing owners: ${e.message}`);
+    }
+    return parsed!.contributors.map(c => c.githubUsername).filter(notUndefined);
```

Comment:
I wonder if we _don't_ use the ./CODEOWNERS from DT in this codebase, looking at this.